main/webapp/modules/core/scripts/views/data-table/menu-edit-cells.js

```diff
@@ -351,10 +353,18 @@ DataTableColumnHeaderUI.extendMenu(function(column, columnHeaderUI, menu) {
           return;
         }
       } else if (mode === "cases") {
-        config.separator = "(?<=[a-z]|[a-z][\\s])(?=[A-Z])";
+        if(elmts.reversTranistionCases[0].checked) {
+          config.separator = "(?<=[A-Z]|[À-Ý]|[A-Z\\s][À-Ý\\s])(?=[a-z]|[à-ÿ])";
+        } else {
+          config.separator = "(?<=[a-z]|[a-z\\s]|[à-ÿ]|[à-ÿ\\s])(?=[A-Z]|[À-Ý])";
```

Comment:
I would avoid hard-coding the character ranges. We should be able to use unicode character ranges as specified here: https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html. It should just be a matter of enabling unicode in the Pattern declaration and then using the corresponding escape sequences.